### PR TITLE
fix find_package

### PR DESCRIPTION
**Pull request by DenisBiryukov91**
fix cmake PacakgeConfig.cmake

### CODE REVIEW

File: zenohpico.pc [hunk 1]
```diff
@@ -1,8 +1,8 @@
-prefix=/usr/local
+prefix=/Users/denisbiryukov/repositories/work/zenoh-c-install
```
Comment (cguimaraes) [inline zenohpico.pc:2]: Revert this line.

File: include/zenoh-pico/api/constants.h [hunk 2]
```diff
@@ -88,8 +88,9 @@ typedef enum {
  *
  * Enumerators:
  *  Z_KEYEXPR_INTERSECTION_LEVEL_DISJOINT: 2 key expression do not intersect.
- *  Z_KEYEXPR_INTERSECTION_LEVEL_INTERSECTS: 2 key expressions intersect, i.e. there exists at least one key expression
- * that is included by both. Z_KEYEXPR_INTERSECTION_LEVEL_INCLUDES: First key expression is the superset of second one.
+ *  Z_KEYEXPR_INTERSECTION_LEVEL_INTERSECTS: 2 key expressions intersect, i.e. there exists at least one key that is
+ * included by both.
+ *  Z_KEYEXPR_INTERSECTION_LEVEL_INCLUDES: First key expression is the superset of second one.
```
Comment (cguimaraes) [inline include/zenoh-pico/api/constants.h:93]: Also, consider this change in a different PR.

File: zenohpico.pc (hunk 1 above)
Comment (DenisBiryukov91) [reply zenohpico.pc]: done

File: include/zenoh-pico/api/constants.h (hunk 2 above)
Comment (DenisBiryukov91) [reply include/zenoh-pico/api/constants.h]: ok, removed from pr

**Review by jean-roland — APPROVED** [review]

**Pull request merged**